src/modules/GroupChannel/context/utils.ts

```diff
@@ -37,5 +37,11 @@ export const isDisabledBecauseMuted = (groupChannel?: GroupChannel | null) => {
 };
 
 export const isDisabledBecauseSuggestedReplies = (channel: GroupChannel | null | undefined, enableSuggestedReplies: boolean) => {
-  return enableSuggestedReplies && !!channel?.lastMessage?.extendedMessagePayload?.disable_chat_input;
+  return enableSuggestedReplies
+      && Array.isArray(channel?.lastMessage?.extendedMessagePayload?.suggested_replies)
+      && !!channel?.lastMessage?.extendedMessagePayload?.disable_chat_input;
+};
+
+export const isDisabledBecauseMessageForm = (channel: GroupChannel | null | undefined) => {
```

Comment:
could you add `config.groupChannel.enableFormTypeMessage` ?

Reply:
@bang9 Currently there is no point for doing that for ai-widge/experimental branch because only widget is using it. This should be done later when we move form message from widget to UIKit (develop).

Reply:
Let me add enableFormTypeMessage in main branch and then rebase ai-widget/experiment onto main

Reply:
Okay!